migrations/V22__fix_circle_check_on_kpi_def_view.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE VIEW public.kpi_definition_with_latest_values AS
```

Comment:
Approved, just a note for future: I can see that there were several migrations for that particular view because of some problems with it. If we want to test something we can apply the migration from a branch by changing the on.push.branches value in .github/migrate.yaml.

**HOWEVER** :D  changes like this one are destructive so we need to be very careful with them. A good idea would be to test it on a separate instance first and save the existing definition.

Reply:
noted ! I will ask you next time as it was indeed a bit challenging to make sure the migration would work